tidy3d/components/heat_charge/simulation.py

```diff
@@ -247,17 +313,22 @@ def names_exist_bcs(cls, val, values):
     def not_all_neumann(cls, val):
         """Error if all boundary conditions are Neumann bc."""
 
-        if len(val) == 0 or all(isinstance(bc_spec.condition, HeatFluxBC) for bc_spec in val):
+        NeumannBCs = (HeatFluxBC, CurrentBC, InsulatingBC)
+        names_neumann_Bcs = [BC.__class__.__name__ for BC in NeumannBCs]
+
+        if len(val) == 0 or all(isinstance(bc_spec.condition, NeumannBCs) for bc_spec in val):
             raise SetupError(
-                "Heat simulation contains only 'HeatFluxBC' (Neumann) boundary conditions. Steady-state solution is undefined in this case."
+                "Your 'HeatChargeSimulation' contains only Neumann-type boundary conditions. "
+                "Steady-state solution is undefined in this case. "
+                f"Current Neumann BCs are {names_neumann_Bcs}"
```

Comment:
Does this take into account situations when one simulation has a dirichlet bc, but the other one is all-neumann? We should make sure that each of simulations is well-defined